testing/run-write-tests.rst

```diff
@@ -86,10 +86,24 @@ if the failures are transient or consistent.
 The ``-uall`` flag allows the use of all available
 resources so as to not skip tests requiring, e.g., Internet access.
 
-To check for reference leaks (only needed if you modified C code), use the
-``-R`` flag.  For example, ``-R 3:2`` will first run the test 3 times to settle
-down the reference count, and then run it 2 more times to verify if there are
-any leaks.
+To check for reference leaks (only needed if you modified C code),
+you can enable reference leak checking during testing using the ``-R`` flag.
+For example, using the command::
+
+    python -m test <test_name> -R :
+
+This default setting performs a few initial warm-up runs to stabilize the reference count,
+followed by additional runs to check for leaks.
+
+If you want more control over the number of runs, you can specify ``warmups`` and ``repeats`` explicitly
+For example::
```

Comment:
Rather than an example, this is a generic command with placeholders.
```suggestion
If you want more control over the number of runs, you can specify ``warmups`` and ``repeats`` explicitly::
```